functional-tests/base-test.py

```diff
@@ -0,0 +1,29 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+# Setting up saucelabs config
+username = os.environ["SAUCE_USERNAME"]
+access_key = os.environ["SAUCE_ACCESS_KEY"]
+capabilities = {
+    'platform': "Mac OS X 10.9",
+    'browserName': "chrome",
+    'version': "31",
+}
+capabilities["tunnel-identifier"] = os.environ["TRAVIS_JOB_NUMBER"]
+hub_url = "%s:%s@localhost:4445" % (username, access_key)
+driver = webdriver.Remote(desired_capabilities=capabilities, command_executor="https://%s/wd/hub" % hub_url)
```

Comment:
This is what we pass to sauce labs (taken from this ex. https://saucelabs.com/blog/run-your-selenium-tests-completely-in-the-cloud-using-travis-ci-and-sauce-labs)